### DISCUSSION TITLE
We need a place to mention allcontributors

### DISCUSSION THREAD

**Discussion by tkoyama010** [bjlittle/geovista#955] [Ideas]
When I commented https://github.com/bjlittle/geovista/discussions/774#discussioncomment-9939168 , allcontributors didn't recognize it, so I had a hard time finding a place to mention it. We need a place to mention allcontributors when someone make some kind of contribution in the discussion.

**Comment by bjlittle** [comment]
Yes, I've stumbled across this problem too.

Also, the bot doesn't scrape Discussion comments, which is a shame as it would be quite convenient. I'll raise an issue with them for this feature. 

In the meantime, it might be useful to have a dedicated pinned open issue where people can comment to the bot. 

@tkoyama010 Does that sound reasonable?

**Reply by tkoyama010** [reply]
Great idea. Thanks!

**Reply by trexfeathers** [reply]
>Also, the bot doesn't scrape Discussion comments, which is a shame as it would be quite convenient. I'll raise an issue with them for this feature.

I think it uses the REST API ([having looked at this](https://github.com/all-contributors/app/blob/86d6bc7baafae823f34e7be2b194c645366ef03e/contributing/bot-architecture.md)). This is understandable as it's much easier than GraphQL. Unfortunately it's also a thing of the past 🦖 - no newer GitHub features are accessible this way (e.g. Discussions or Projects V2).

**Comment by bjlittle** [comment]
@tkoyama010 We can now use issue #959 to make ad-hoc contributor nominations. We'll leave it open and pinned as a community convenience.

I'm going to close this discussion now, thanks :+1:
